Ellen Homework 1.py

```diff
@@ -0,0 +1,31 @@
+import time 
+import random 
+
+
+random_creatures = ["gorgon", "vampire", "gruffalo", "zombie", "minatour"]
+
+
+def greet_user(username):
+    print("Hello, " + username + "!")
+    
+greet_user('arcade')
+greet_user('ellen')
+greet_user('sophie')
+

```

Comment:
If you wanted you could add this function inside the newly created intro function.